Case-sensitive WFM name matching may cause silent lookup failures

## Description

The Warframe Market (WFM) item name lookups in `src/Generator.ts` use case-sensitive equality checks, which may produce silent misses if WFM v2 item names differ in casing from WFCD reward names.

## Current Behavior

- Line 122: `this.wfmItems?.data.find((x) => x.i18n.en.name === rawReward.itemName)`
- Line 156: `this.wfmItems?.data.find((x) => x.i18n.en.name === \`${name.trim()} Relic\`)`

Both use strict equality (`===`) for name comparison.

## Inconsistency

Other WFCD item lookups in the same file (lines 134, 144) explicitly use `.toLowerCase()` for matching, suggesting case-insensitive comparison is the intended pattern.

## Suggested Fix

Apply case-insensitive matching consistently across all WFM lookups:

```typescript
const wfmInfo = this.wfmItems?.data.find(
  (x) => x.i18n?.en?.name?.toLowerCase() === rawReward.itemName.toLowerCase()
);
```

## Context

This is **pre-existing behavior** noted during review of PR #366.

- PR: https://github.com/WFCD/warframe-relic-data/pull/366
- Comment: https://github.com/WFCD/warframe-relic-data/pull/366#discussion_r2837018373
- Requested by: @TobiTenno

@TobiTenno Any Tips? I only did npm i after updating my fork.
```
> @wfcd/relics@2.0.26 start
> npm run build && node dist/Build.js


> @wfcd/relics@2.0.26 build
> tsdown ./src/index.ts ./src/Build.ts

ℹ tsdown v0.20.3 powered by rolldown v1.0.0-rc.3
ℹ config file: /home/tizian/GitHub/warframe-relic-data/tsdown.config.json 
ℹ entry: src/index.ts, src/Build.ts
ℹ target: node14
ℹ tsconfig: tsconfig.json
ℹ Build start
ℹ Cleaning 11 files
ℹ [CJS] dist/Build.cjs                     0.87 kB │ gzip: 0.36 kB
ℹ [CJS] dist/index.cjs                     0.26 kB │ gzip: 0.17 kB
ℹ [CJS] dist/VersionManager-DkHKfTUL.cjs  12.28 kB │ gzip: 3.56 kB
ℹ [CJS] 3 files, total: 13.42 kB
ℹ [CJS] dist/index.d.cts  5.47 kB │ gzip: 1.67 kB
ℹ [CJS] dist/Build.d.cts  0.01 kB │ gzip: 0.03 kB
ℹ [CJS] 2 files, total: 5.49 kB
✔ Build complete in 488ms
ℹ [ESM] dist/Build.mjs                     0.78 kB │ gzip: 0.40 kB
ℹ [ESM] dist/index.mjs                     0.12 kB │ gzip: 0.10 kB
ℹ [ESM] dist/VersionManager-CGY3TWJF.mjs  10.23 kB │ gzip: 3.03 kB
ℹ [ESM] dist/chunk-Dco9itRc.mjs            0.20 kB │ gzip: 0.17 kB
ℹ [ESM] dist/index.d.mts                   5.51 kB │ gzip: 1.71 kB
ℹ [ESM] dist/Build.d.mts                   0.01 kB │ gzip: 0.03 kB
ℹ [ESM] 6 files, total: 16.87 kB
✔ Build complete in 489ms
node:internal/modules/cjs/loader:1386
  throw err;
  ^

Error: Cannot find module '/home/tizian/GitHub/warframe-relic-data/dist/Build.js'
    at Function._resolveFilename (node:internal/modules/cjs/loader:1383:15)
    at defaultResolveImpl (node:internal/modules/cjs/loader:1025:19)
    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1030:22)
    at Function._load (node:internal/modules/cjs/loader:1192:37)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:237:24)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:171:5)
    at node:internal/main/run_main_module:36:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v22.19.0
```

--- a/src/Generator.ts
+++ b/src/Generator.ts
@@ -119,7 +119,9 @@ export class Generator {
     const rewards = rawRelic.rewards.map((rawReward) => {
       const { chance } = rawReward;
       const { rarity } = rawReward;
-      const wfmInfo = this.wfmItems?.data.find((x) => x.i18n.en.name === rawReward.itemName);
+      const wfmInfo = this.wfmItems?.data.find((x) => {
+        return x.i18n.en.name.toLowerCase() === rawReward.itemName.toLowerCase();
+      });
       const isSpecial = ['Forma', 'Kuva', 'Exilus', 'Riven'].find((x) =>
         // eslint-disable-next-line @typescript-eslint/comma-dangle
         rawReward.itemName.toLowerCase().includes(x.toLowerCase())
@@ -153,7 +155,9 @@ export class Generator {
       });
     }
 
-    const wfm = this.wfmItems?.data.find((x) => x.i18n.en.name === `${name.trim()} Relic`);
+    const wfm = this.wfmItems?.data.find((x) => {
+      return x.i18n.en.name.toLowerCase() === `${name.trim()} Relic`.toLowerCase();
+    });
     if (!wfm) {
       logger.error(`Failed to get relic item from wfm: ${name}`);
     }

